test/integration/go.mod

```diff
@@ -0,0 +1,33 @@
+module github.com/terraform-google-modules/terraform-google-project-factory/test/integration
+
+go 1.17
+
+require (
+	github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test v0.0.0-20210818060910-07c4cb2d2107
```

Comment:
Lets use latest release. You can update via `go get -u github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test`
https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test?tab=versions